metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -174,12 +176,19 @@ def _build_base_metric_output_node(
         """Builds a node to compute a metric that is not defined from other metrics."""
         metric_reference = metric_spec.reference
         metric = self._metric_lookup.get_metric(metric_reference)
-        metric_input_measure_specs = self._measures_for_metric(
+        metric_input_measure_spec = self._build_metric_input_measure_for_base_metric(
             metric_reference=metric_reference,
             column_association_resolver=self._column_association_resolver,
+            query_contains_metric_time=queried_linkable_specs.contains_metric_time,
+            child_metric_offset_window=metric_spec.offset_window,
+            child_metric_offset_to_grain=metric_spec.offset_to_grain,
+            culmination_description=MeasureCulminationDescription(
```

Comment:
Agree with @courtneyholcomb, this name is odd. I'd go with:

`cumulative_description=CumulativeMeasureDescription`

Or else

`accumulation_description=MeasureAccumulationDescription`

Reply:
Updated.